Microsoft.Toolkit.Uwp.UI.Controls/Loading/Loading.xaml

```diff
@@ -0,0 +1,112 @@
+                                            <EasingDoubleKeyFrame KeyTime="0:0:0.3" Value="40">
+                                                <EasingDoubleKeyFrame.EasingFunction>
+                                                    <QuadraticEase EasingMode="EaseInOut"/>
+                                                </EasingDoubleKeyFrame.EasingFunction>
+                                            </EasingDoubleKeyFrame>
+                                        </DoubleAnimationUsingKeyFrames>
+                                        <DoubleAnimationUsingKeyFrames 
+                                            Storyboard.TargetProperty="Opacity" 
+                                            Storyboard.TargetName="RootGrid">
+                                            <EasingDoubleKeyFrame KeyTime="0:0:0" Value="1">
+                                                <EasingDoubleKeyFrame.EasingFunction>
+                                                    <QuadraticEase EasingMode="EaseInOut"/>
+                                                </EasingDoubleKeyFrame.EasingFunction>
+                                            </EasingDoubleKeyFrame>
+                                            <EasingDoubleKeyFrame KeyTime="0:0:0.3" Value="0">
+                                                <EasingDoubleKeyFrame.EasingFunction>
+                                                    <QuadraticEase EasingMode="EaseInOut"/>
+                                                </EasingDoubleKeyFrame.EasingFunction>
+                                            </EasingDoubleKeyFrame>
+                                        </DoubleAnimationUsingKeyFrames>
+                                        <ObjectAnimationUsingKeyFrames
+                                            Storyboard.TargetName="RootGrid"
+                                            Storyboard.TargetProperty="Visibility">
+                                            <DiscreteObjectKeyFrame KeyTime="0:0:0.3">
+                                                <DiscreteObjectKeyFrame.Value>
+                                                    <Visibility>Collapsed</Visibility>
+                                                </DiscreteObjectKeyFrame.Value>
+                                            </DiscreteObjectKeyFrame>
+                                        </ObjectAnimationUsingKeyFrames>
+                                    </Storyboard>
+                                </VisualState>
+                            </VisualStateGroup>
+                        </VisualStateManager.VisualStateGroups>
+                        <Grid x:Name="BackgroundGrid" Background="{TemplateBinding LoadingBackground}" Opacity="{TemplateBinding LoadingOpacity}"/>
+                        <ContentPresenter x:Name="ContentGrid" HorizontalContentAlignment="{TemplateBinding LoadingHorizontalAlignment}" VerticalContentAlignment="{TemplateBinding LoadingVerticalAlignment}">
```

Comment:
Template Bind Horizontal/VerticalAlignment To Horizontal/VerticalContentAlignment